addon/components/lazy-render-wrapper.js

```diff
@@ -102,7 +103,7 @@ export default Ember.Component.extend({
 
 			return true;
 
-		} else if (this.get('isShown') || this.get('tooltipIsVisible')) {
+		} else if (!this.get('disabled') && (this.get('isShown') || this.get('tooltipIsVisible'))) {
 
 			this.set('_hasRendered', true);
 			return true;
```

Comment:
I can't comment below this line so see lines....

111: what if `enableLazyRendering=false` and `disabled=true`?

129:  what if `enableLazyRendering=false` but `_shouldRender=false`? The didInsertElement will still apply $parent event handling and could still render the tooltip, even if it's never shown (not clean)